src/foam/nanos/theme/Themes.js

```diff
@@ -102,6 +102,10 @@ Later themes:
               theme = theme && theme.merge(groupTheme) || groupTheme;
               break;
             }
+            if ( groupTheme ) {
```

Comment:
How does this 'fix' your issue?
Above, we merge themes as we traverse the parents.   
Also, this replaces any merging we've already done.